examples/dbapi.ipynb

```diff
@@ -410,17 +422,24 @@
    "cell_type": "raw",
```

Comment:
let's change this to `markdown` while we at it. Otherwise it's different from other headings.